ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml

```diff
@@ -590,6 +590,19 @@ tests:
     test:
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-aws-ipi
+- as: e2e-osd-ccs-nightly-aws
+  cron: 0 8 * * 3
+  steps:
+    cluster_profile: aws-qe
+    env:
+      CHANNEL_GROUP: nightly
+      E2E_RUN_TAGS: '@aws-ipi and @osd and @network-openshiftsdn and not @fips'
+      OPENSHIFT_VERSION: "4.11"
+      TAG_VERSION: '@4.11'
+      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~DisconnectedOnly&;OSD&
```

Comment:
@yasun1 as we ever talked, we have OSD and CCS label for golang. for this cluster, which is used? just double confirm it.
how to determine which is used per workflow?

cc @jianzhangbjz

Reply:
The cluster here I provide is an 'osd-ccs' cluster. The 'osd-ccs' means it is an osd cluster created with the customer's cloud account. At here, I missed add `ccs` label.